click-to-load/tests.json

```diff
@@ -0,0 +1,72 @@
+{
+    "requestBlocking": {
+        "name": "Request Blocking",
+        "desc": "Requests to a tracker should be blocked or surrogated according to the specification in config",
+        "tests": [
+            {
+                "name": "tracker request should be blocked",
+                "siteURL": "https://example.com/",
+                "requestURL": "https://tracker.test/track.js",
+                "userUnblocked": false,
+                "expectAction": "block",
+                "exceptPlatforms": []
+            },
+            {
+                "name": "same domain - tracker request should not be blocked",
+                "siteURL": "https://tracker.test/",
+                "requestURL": "https://tracker.test/track.js",
+                "userUnblocked": false,
+                "expectAction": "ignore",
+                "exceptPlatforms": []
+            },
+            {
+                "name": "same entity - tracker request should not be blocked",
```

Comment:
Would be nice to do the same entity check with a domain not in the domains list in the CTP configuration.

Reply:
so like: requestURL to tracker.newdomain that belongs to Tracker INC, but is not mentioned in the CTL config?

Reply:
Yea, exactly. How about we add `tracker.org` to `click-to-load/tracker_radar_reference.json` in the Tracker INC entity's domain list, make no changes to the CTP configuration, and then change this test case to have a `siteURL` that uses `tracker.org` instead of `tracker.net`?